cypress/integration/group2/workflowDetails.ts

```diff
@@ -114,14 +114,18 @@ describe('Dockstore Workflow Details', () => {
     });
 
     it('Change to fullscreen and back', () => {
+      cy.get('[data-cy=dag-holder]').should('have.class', 'small');
+      cy.get('[data-cy=dag-holder]').should('not.have.class', 'big');
       cy
         .get('#dag_fullscreen')
-        .click()
-        .get('#dag-col')
-        .should('have.class', 'fullscreen-element')
-        .get('#dag_fullscreen')
-        .click()
-        .should('not.have.class', 'fullscreen-element');
+        .click();
+        // Cypress or electron can't initiate fullscreen because:
```

Comment:
comment on when this would get reactivated if you want to keep this commented code